pyproject.toml

```
@@ -57,40 +58,61 @@ changelog = "https://github.com/Project-OMOTES/simulator-core/blob/main/CHANGELO
 
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools ~= 68.2.2"]
+requires = [
+    "setuptools ~= 69.0.3",
+    "wheel ~= 0.40.0",
+    "setuptools-git-versioning<2",
+]
+
+[tool.setuptools-git-versioning]
+enabled = true
+version_file = "VERSION"
+count_commits_from_version_file = true
+dev_template = "{tag}.dev{ccount}"
+dirty_template = "{tag}.dev{ccount}"
+
 
-[tools.setuptools]
-packages = ["src/simulator_core"]
+# [tool.setuptools]
+# packages = ["src/simulator_core"]
 
+[tool.pytest.ini_options]
+addopts = "--cov=simulator_core --cov-report html --cov-report term-missing --cov-fail-under 80"
```

Werkt dit op Visual Studio Code?

---

Ik had zelf nog een .vscode/launch.json nodig (met mijn versie):

```suggestion
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python Debugger: Current File",
            "type": "debugpy",
            "request": "launch",
            "program": "${file}",
            "console": "integratedTerminal",
            "env": {
                "PYTEST_ADDOPTS": "--no-cov"
            }
        }
    ]
}
```

---

Ik heb mijn launch & settings.json erbij gezet.    test-discover, run en debug werkt zonder issues met vscode

---

Jouw suggestion is niet nodig met de aangepaste settings.json

---

Oke gezien